### PR TITLE
Feature card miniature and api fetch list

### PR DESCRIPTION
**Pull request by Pyramate**
- Created the component movie Card to display a movie info
- Imported Chakra UI to test the library and style components
- Imported moment to display release date in a better format
- Created the component MovilCarouse that takes an URL as props to fetch data and display movie card accordingly
- tested the component on HomePage (can be dropped after testing)

### CODE REVIEW

File: backend/.eslintrc.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "es2021": true
   },
   "extends": ["airbnb-base", "plugin:prettier/recommended"],
+
```
Comment (lchastanet) [inline backend/.eslintrc.json:7]: Empty line !!

File: frontend/.eslintrc.json [hunk 2]
```diff
@@ -18,6 +18,8 @@
   "plugins": ["react", "import"],
   "rules": {
     "no-console": "off",
+    "react/prop-types": 0,
+
```
Comment (lchastanet) [inline frontend/.eslintrc.json:22]: Empty line !

File: backend/package.json [hunk 3]
```diff
@@ -14,8 +14,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@chakra-ui/react": "^1.8.8",
+    "@emotion/react": "^11.9.0",
+    "@emotion/styled": "^11.8.1",
```
Comment (lchastanet) [inline backend/package.json:19]: Not required in backend

File: backend/package.json [hunk 4]
```diff
@@ -14,8 +14,12 @@
...
     "dotenv": "^10.0.0",
     "express": "^4.17.2",
+    "framer-motion": "^6.2.10",
```
Comment (lchastanet) [inline backend/package.json:22]: Not required in backend

**Review by lchastanet — CHANGES_REQUESTED** [review]
Take a look on few changes and it should be ok !

File: backend/package.json (hunk 3 above)
Comment (Pyramate) [reply backend/package.json]: fixed

File: frontend/.eslintrc.json (hunk 2 above)
Comment (Pyramate) [reply frontend/.eslintrc.json]: ok

**Review by lchastanet — APPROVED** [review]
Nickel

**Pull request merged**